packages/nodes-base/nodes/ReadPdf/ReadPDF.node.ts

```diff
@@ -2,6 +2,7 @@ import { IExecuteFunctions } from 'n8n-core';
 
 import { INodeExecutionData, INodeType, INodeTypeDescription } from 'n8n-workflow';
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
needed for disabling rule

Reply:
The exception is needed if you want to use `require` but the exception should not be needed because using `require` should not be needed. Why cannot we `import`?